lib/util.py

```diff
@@ -590,7 +591,7 @@ def create_URI(addr, amount, message):
     if amount:
         query.append('amount=%s'%format_satoshis_plain(amount))
     if message:
-        if type(message) == unicode:
+        if six.PY2 and type(message) == unicode:
```

Comment:
`unicode` is only defined in Python 2, so make sure it doesn't get evaluated in Python 3.